avroconsumer/src/main/java/com/epam/learning/backendservices/AvroConsumer.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.learning.backendservices;
+
+import com.epam.learning.backendservices.kafka.BuiRegistration;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Logger;
+
+public class AvroConsumer {
+
+    private static final Logger logger = Logger.getLogger(AvroConsumer.class.getName());
+
+    public static void main(String[] args) throws IOException {
+        InputStream inputStream = AvroConsumer.class.getClassLoader().getResourceAsStream("avroconsumer.properties");
+        Properties properties = new Properties();
+        properties.load(inputStream);
```

Comment:
The Javadoc of load method tells: "The specified stream remains open after this method returns". Which means you are responsible for closing that stream. Use try with resources or manually invoke close method after you have loaded properties.

Reply:
Done